providers/aws/resources/aws_vpc.go

```diff
@@ -546,11 +546,16 @@ func (a *mqlAwsVpc) routeTables() ([]interface{}, error) {
 			if err != nil {
 				return nil, err
 			}
+			dictAssociations, err := convert.JsonToDictSlice(routeTable.Associations)
+			if err != nil {
+				return nil, err
+			}
 			mqlRouteTable, err := CreateResource(a.MqlRuntime, "aws.vpc.routetable",
 				map[string]*llx.RawData{
-					"id":     llx.StringDataPtr(routeTable.RouteTableId),
```

Comment:
@mm-weber is some of your code missing? I don't see the associations resource here?